src/components/CommentComponent.vue

```diff
@@ -0,0 +1,31 @@
+<script>
+  export default {
+    name: 'CommentComponent',
+    emits: ['delete'],
+    props: {
+      comment: Object,
+    },
+    methods: {
+      handlerDeleteComment(commentId) {
+        this.$emit('delete', commentId);
+      },
+    },
+  }
+</script>
+
+<template>
+  <article class="message is-small">
+    <div class="message-header">
+      <a :href="`mailto:${comment.email}`">{{ comment.name }}</a>
```

Comment:
Let's move this to a variable to improve readability `"`mailto:${comment.email}`"` 

Also use destruction assignment to a comment object